ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -590,6 +932,195 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Openings 1",
+                "\n",
+                "\nImport of Scenario Actuals where DataSource contains openings for scenario and best estimate."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var basicIfrsVariable = new IfrsVariable{Partition = partition.Id, DataNode = groupOfInsuranceContracts, AccidentYear = null, AmountType = AmountTypes.PR, Novelty = Novelties.C, EstimateType = EstimateTypes.AA};",
+                "\n",
+                "\nvar inputDataSetForDataSource = new IfrsVariable[]{",
+                "\n    basicIfrsVariable with {AocType = AocTypes.BOP, EstimateType = EstimateTypes.DA, Value = 1000.0},",
+                "\n    basicIfrsVariable with {AocType = AocTypes.CL, Value = 150.0},",
+                "\n    basicIfrsVariable with {AocType = AocTypes.BOP, Novelty = Novelties.I, Value = 90.0},",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AocType = AocTypes.BOP, Novelty = Novelties.I, Value = 90.7},",
+                "\n    basicIfrsVariable with {Partition = previousPeriodPartition.Id, AocType = AocTypes.EOP, Novelty = Novelties.C, Value = 89.5},",
+                "\n    basicIfrsVariable with {Partition = previousPeriodPartitionScenarioMTUP.Id, AocType = AocTypes.EOP, Novelty = Novelties.C, Value = 89.1}",
+                "\n};",
+                "\n",
+                "\nvar inputDataSetForWorkspace = new IfrsVariable[]{",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AocType = AocTypes.CL, Value = -15.0},",
```

Comment:
Is CL,AA a valid input? why not a CF,A?

Reply:
the logic to be tested is identical for both options.

Reply:
done